scripts/safe_rename.js

```diff
@@ -10,8 +10,9 @@ const log = (...args) => {
     }
 }
 
+/** Return date (without time) in ISO format. */
 const formatDate = (date) => {
-    return `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`
+    return date.toISOString().slice(0, 10)
```

Comment:
@Twixes - worth adding a test for this? If nothing else, it clearly indicates to someone reading the code what format we're expecting the date to come back in.

Reply:
Sure